.env.local.default

```diff
@@ -193,3 +193,7 @@ HARPGL_API_KEY=""
 #define user scope 
 DEFAULT_GUEST_SCOPES=editor:write,location:read,location:write
 DEFAULT_USER_SCOPES=editor:write,location:read,location:write
+
+#define Blockchain url
+BLOCKCHAIN_URL = http://blockchain.xrengine.io/api/v1
```

Comment:
we should use the hostname that a local instance of biab would run on, rather than point at our official server

Reply:
changes done